frame_name: base_laser_link
angle_min: -3.141592
angle_max: 3.141592
angle_increment: 0.005
range_min: 0.025
range_max: 60
